modules/templates/helper.go

```diff
@@ -620,6 +622,17 @@ func Str2html(raw string) template.HTML {
 	return template.HTML(markup.Sanitize(raw))
 }
 
+// Markdown2html render Markdown text to HTML (non-sanitize)
+func Markdown2html(raw string) template.HTML {
+	var err error
+	var renderedContent string
+	if renderedContent, err = markdown.RenderString(&markup.RenderContext{}, raw); err != nil {
+		log.Warn("Markdown2html: Invalid markdown? %v", err)
+		return template.HTML(markup.Sanitize(raw)) // this was gitea 1.15.* behaviour
```

Comment:
The default behavior is `template.HTML(html.EscapeString(raw))`

As we discussed before, the `Str2html` is quite misleading, let's forget it.

Reply:
ok, so let's roll with `template.HTML(html.EscapeString(raw))`